crates/bazed-core/src/buffer.rs

```diff
@@ -85,6 +85,20 @@ impl Buffer {
                 .expect("Caret stored in BufferRegions was not a valid offset into the buffer")
         })
     }
+
+    // Great naming... better than all_caret_coordinate_regions
```

Comment:
I'm not sure this is better, also I'm not sure that's the proper place to make fun of elk.

Reply:
Note: that was not the previous name. there was no previous name.

Reply:
fine